src/engine/strat_engine/filesystem.rs

```diff
@@ -147,39 +149,14 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
-pub fn mount_fs(dev_path: &Path, mount_point: &Path) -> EngineResult<()> {
-
-    debug!("Mount filesystem {:?} on : {:?}", dev_path, mount_point);
-    let output = try!(Command::new("mount")
-                          .arg(&dev_path)
-                          .arg(mount_point)
-                          .output());
-
-    if output.status.success() {
-        debug!("Mounted filesystem on {:?}", mount_point)
-    } else {
-        let message = String::from_utf8_lossy(&output.stderr);
-        debug!("stderr: {}", message);
-        return Err(EngineError::Engine(ErrorEnum::Error, message.into()));
-    }
-    Ok(())
-}
-
-pub fn unmount_fs<I, S>(mount_point: &Path, flags: I) -> EngineResult<()>
-    where I: IntoIterator<Item = S>,
-          S: AsRef<OsStr>
+/// Mount the filesystem located at devnode at mountpoint.
+pub fn mount_fs<P1, P2>(devnode: &P1, mountpoint: &P2) -> EngineResult<()>
```

Comment:
Feels strange to have `mount_fs` with no `unmount_fs`. Do we really need `mount_fs` any more? Can MDV just use the nix method directly? Or if we do want `mount_fs`, then we need a trivial `unmount_fs` wrapping umount, just for the sake of consistency 😃